platform/darwin/src/NSValue+MGLStyleEnumAttributeAdditions.h

```diff
@@ -0,0 +1,150 @@
+// This file is generated.
+// Edit platform/darwin/scripts/generate-style-code.js, then run `make style-code-darwin`.
+
+#import <Foundation/Foundation.h>
+
+#import "MGLFillStyleLayer.h"
+#import "MGLLineStyleLayer.h"
+#import "MGLSymbolStyleLayer.h"
+#import "MGLCircleStyleLayer.h"
+#import "MGLRasterStyleLayer.h"
+#import "MGLBackgroundStyleLayer.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface NSValue (MGLStyleEnumAttributeAdditions)
```

Comment:
Run `make idocument` and verify that the sole documented NSValue category isn’t called `MGLStyleEnumAttributeAdditions`.

The existing documented category should be called `MGLAdditions`, not `MGLGeometryAdditions`. While you’re at it, can you remove the rump `MGLGeometryAdditions` category in MGLGeometry.h?